front/src/common/RollingStockSelector/RollingStockCardDetail.tsx

```diff
@@ -40,34 +56,42 @@ export default function RollingStockCardDetail(props) {
       };
       // AC & HEATING curves (optional)
       curvesData.modes[mode].curves.forEach((curve) => {
-        const serieId = `${name} ${curve.cond.comfort}`;
-        transformedCurves[serieId] = { ...curve.curve, mode: name, comfort: curve.cond.comfort };
+        if (curve.cond) {
+          const serieId = `${name} ${curve.cond.comfort}`;
```

Comment:
```suggestion
          const optionalCurveName = `${name} ${curve.cond.comfort}`;
```
Can you rename this variable ? (suggestion)

Reply:
done